src/ai/susi/mind/SusiSkill.java

```diff
@@ -299,6 +301,10 @@ else if (bang_type.equals("console")) {
                     if(skillName.length() > 0)
                         json.put("skill_name",skillName);
                 }
+                if (line.startsWith("::private_skill") && (thenpos = line.indexOf(' ')) > 0) {
```

Comment:
just call it "private", not "private_skill" because all modifiers change status of the skill somehow.

Reply:
Okay